packages/strapi-admin/admin/src/containers/App/tests/actions.test.js

```diff
@@ -0,0 +1,141 @@
+import {
+  FREEZE_APP,
+  GET_APP_PLUGINS_SUCCEEDED,
+  LOAD_PLUGIN,
+  PLUGIN_DELETED,
+  PLUGIN_LOADED,
+  UNFREEZE_APP,
+  UNSET_HAS_USERS_PLUGIN,
+  UPDATE_PLUGIN,
+} from '../constants';
+import {
+  freezeApp,
+  getAppPluginsSucceeded,
+  loadPlugin,
+  pluginDeleted,
+  pluginLoaded,
+  unfreezeApp,
+  unsetHasUserPlugin,
+  updatePlugin,
+} from '../actions';
+
+describe('<App /> actions', () => {
+  describe('freezeApp', () => {
+    it('should return the correct type and the passed data', () => {
+      const data = { strapi: 'isCool' };
+      const expected = {
+        type: FREEZE_APP,
+        data,
+      };
+
+      expect(freezeApp(data)).toEqual(expected);
+    });
+  });
+
+  describe('unfreezeApp', () => {
+    it('should return the correct type', () => {
```

Comment:
Same here?

Reply:
It wrapped in a describe so I believe that the naming is correct.